k8s/apis/model/v1/common_types.go

```diff
@@ -8,9 +8,18 @@ package v1
 
 import (
 	"github.com/eclipse-symphony/symphony/api/pkg/apis/v1alpha1/model"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const (
+	// ReconciliationPolicy_Periodic allows the controller to reconcile periodically
+	ReconciliationPolicy_Periodic ReconciliationPolicy = "periodic"
```

Comment:
We only have one policy - periodic now. And based on our previous API spec review plan, we will define the reconciliation policy as "enabled" or "disabled", this will be changed in future PR. This PR is just to align with AIO changes.

Reply:
I have changed the reconciliation policy to align with api spec. Please see the description in doc: https://github.com/eclipse-symphony/symphony/blob/042f2d8c33db688c2de845f3616b86995db11105/docs/symphony-book/build_deployment/symphony_mode.md#state-reconciliation-flows.